helmfile/values/falco/falco-common.yaml.gotmpl

```diff
@@ -54,263 +57,36 @@ falcoctl:
-          )
-        )
-    - macro: allowed_clear_log_files
-      condition: (
-          (fd.name startswith "/var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/" and proc.cmdline = container) or
-          (fd.name glob "/var/lib/containerd/tmpmounts/containerd-mount*/var/log/dpkg.log" and proc.cmdline = containerd) or
-          (fd.name glob "/var/lib/containerd/tmpmounts/containerd-mount*/var/log/yum.log" and proc.cmdline = containerd) or
-          (fd.name glob "/var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/snapshots/*/fs/var/log/yum.log" and proc.cmdline = containerd) or
-          (fd.name glob "/var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/snapshots/*/fs/var/log/dpkg.log" and proc.cmdline = containerd)
-        )
-    - macro: user_shell_container_exclusions
-      condition: (
-          (container.image.repository = registry.opensource.zalan.do/acid/spilo-14 and proc.cmdline glob 'sh -c envdir "/run/etc/wal-e.d/env" /scripts/restore_command.sh *') or
-          (container.image.repository = docker.io/library/rabbitmq and proc.cmdline = 'sh -c "/usr/local/lib/erlang/erts-13.0.4/bin/epmd" -daemon')
-        )
-    - macro: user_known_shell_config_modifiers
-      condition: (
-          (fd.name startswith "/var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/" and proc.cmdline = containerd) or
-          (fd.name glob "/var/lib/containerd/tmpmounts/containerd-mount*/root/.bashrc" and proc.cmdline = containerd) or
-          (fd.name glob "/var/lib/containerd/tmpmounts/containerd-mount*/root/.profile" and proc.cmdline = containerd) or
-          (fd.name glob "/var/lib/containerd/tmpmounts/containerd-mount*/etc/skel/.bash_logout" and proc.cmdline = containerd) or
-          (fd.name glob "/var/lib/containerd/tmpmounts/containerd-mount*/etc/skel/.bashrc" and proc.cmdline = containerd) or
-          (fd.name glob "/var/lib/containerd/tmpmounts/containerd-mount*/etc/skel/.profile" and proc.cmdline = containerd) or
-          (fd.name glob "/var/lib/containerd/tmpmounts/containerd-mount*/etc/skel/.bash_profile" and proc.cmdline = containerd)
-        )
-    - macro: user_expected_terminal_shell_in_container_conditions
-      condition: (
-        (container.image.repository = ghcr.io/zalando/spilo-15 and proc.cmdline = bash)
-        )
-    - macro: user_known_ingress_remote_file_copy_activities
-      condition: (
-          proc.cmdline = 'curl --retry 3 -w %{http_code} -o /dev/null -sk https://localhost/healthz'
-        )
-    - list: known_binaries_to_read_environment_variables_from_proc_files
-      append: true
-      items: [systemd-run, rook, udevadm]
-    - list: known_drop_and_execute_containers
+
+    - rule: Run shell untrusted
       append: true
-      items:
-        - ghcr.io/elastisys/calico-accountant # when .calicoAccountant.backend = nftables
+      condition: and not k8s_containers
```

Comment:
Instead of modifying the rule I would like you to use the macro `user_shell_container_exclusions` that is already part of the rule.
See https://github.com/falcosecurity/rules/blob/e206c1a3f12c310c38afcf6af9c311c312791804/rules/falco_rules.yaml#L601C13-L601C44